package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "memory-fs",
-  "version": "0.3.0",
+  "version": "0.3.1",
```

Comment:
please don't touch the version in PRs. could you remove this line change? Elsewise the PR is fine.